### DISCUSSION TITLE
Different color temperature ranges/scales in a group?

### DISCUSSION THREAD

**Discussion by hyperion007** [Koenkk/zigbee2mqtt#20232] [General]
Hi,

Does anyone have any suggestion on how to handle a group of light/controllers that have different ranges/scales?

For example it a group consists of one IKEA LED1732G11, one GLEDOPTO GL-C-008P and one GLEDOPTO GL-C-006S

Color temperature ranges of these three devices:
The IKEA LED1732G11: 250 - 454
GLEDOPTO GL-C-008P: 158 - 495
GLEDOPTO GL-C-006S: 150 - 500

I want to be able to adjust the color temperature for the group by sending a command to the group and have all the devices in the group "match".

Is there a way to specify for each device to have the same scale? 
I'm thinking along the lines of:
Settings (specific)
map(deviceColorTemperature, deviceColorTempMin, deviceColorTempMax, 0, 254) true/false

Thanks!

**Comment by sjorge** [comment]
colorTemperature is a value in mireds, in theory a value of 370 for example should be the same on all of those. (In practice there are small variations between different manufacturers)

If you stick to a value that falls within the largest min and smallest max, you should be same. In your case 250 - 454

**Reply by hyperion007** [reply]
Ok, I guess I'll use that where possible and use the hard coded min/max values for specific devices where needed.
